io.github.alexays.waybar.yaml

```diff
@@ -0,0 +1,202 @@
+app-id: io.github.alexays.waybar
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.vala
+command: waybar
+finish-args:
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+modules:
+  - name: libsigc++
+    buildsystem: meson
+    build-options:
+      config-opts:
+        - -Dbuild-examples=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libsigc++/2.10/libsigc++-2.10.8.tar.xz
+        sha256: 235a40bec7346c7b82b6a8caae0456353dc06e71f14bc414bcc858af1838719a
+
+  - name: glibmm
+    buildsystem: meson
+    build-options:
+      config-opts:
+        - -Dbuild-examples=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.7.tar.xz
+        sha256: fe02c1e5f5825940d82b56b6ec31a12c06c05c1583cfe62f934d0763e1e542b3
+
+  - name: cairomm
+    buildsystem: meson
+    build-options:
+      config-opts:
+        - -Dbuild-examples=false
+    sources:
+      - type: archive
+        url: https://www.cairographics.org/releases/cairomm-1.14.5.tar.xz
+        sha256: 70136203540c884e89ce1c9edfb6369b9953937f6cd596d97c78c9758a5d48db
+
+  - name: pangomm
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/pangomm/2.46/pangomm-2.46.4.tar.xz
+        sha256: b92016661526424de4b9377f1512f59781f41fb16c9c0267d6133ba1cd68db22
+
+  - name: atkmm
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.4.tar.xz
+        sha256: 0a142a8128f83c001efb8014ee463e9a766054ef84686af953135e04d28fdab3
+
+  - name: gtkmm
+    buildsystem: meson
+    build-options:
+      config-opts:
+        - -Dbuild-demos=false
+        - -Dbuild-tests=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gtkmm/3.24/gtkmm-3.24.9.tar.xz
+        sha256: 30d5bfe404571ce566a8e938c8bac17576420eb508f1e257837da63f14ad44ce
+
+  - name: chrono-date
+    buildsystem: cmake-ninja
+    build-options:
+      config-opts:
+        - -DBUILD_TZ_LIB=ON
+        - -DUSE_SYSTEM_TZ_DB=ON
+        - -DENABLE_DATE_TESTING=OFF
+        - -DCMAKE_BUILD_TYPE=None
+        - -DBUILD_SHARED_LIBS=ON
+    sources:
+      - type: archive
+        url: https://github.com/HowardHinnant/date/archive/v3.0.1.tar.gz
+        sha256: 7a390f200f0ccd207e8cff6757e04817c1a0aec3e327b006b7eb451c57ee3538
+
+  - name: fmt
+    buildsystem: cmake-ninja
+    build-options:
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=None
```

Comment:
is `None` really a valid type? Usually this is `Release` or `RelWithDebInfo`

Reply:
Release sounds much more reasonable.

Reply:
I added release build to the meson packages too. (Unlikely to have any impact on an app like Waybay...)